components/brave_ads/browser/ads_service_factory.cc

```diff
@@ -97,6 +97,7 @@ void AdsServiceFactory::RegisterProfilePrefs(
 
   #if defined(OS_ANDROID)
     registry->RegisterUint64Pref(prefs::kAdsPerDay, 12);
+    registry->RegisterUint64Pref(prefs::kAdsPerSameTime, 3);
```

Comment:
Please remove registry entry and change to `static const int kMaximumAdNotificationsInAndroidNotificationDrawer = 3;` in `static_values.h` as this value cannot be changed via the UI so should not be exposed

Reply:
@tmancey 
My initial thought was the same. But I followed pattern `kAdsPerDay` which is not exposed either.
Why we don't leave it in preferences? I think, it's very good candidate to be potentially exposed to ads UI settings.

Reply:
@gdregalo by leaving in it would mean iOS also have to make changes which will delay their release. `kAdsPerDay` was legacy as in Muon a user could change this value from the UI